### PR TITLE
Transforms cannot load Plugins

### PR DESCRIPTION
**Pull request by gokulavasan**
Currently Transforms have access to AdapterContext but not to AdapterPluginRegistry. Removing AdapterContext access in Transforms so that Transform Plugins cannot load external Plugins in 3.0.

JIRA : https://issues.cask.co/browse/CDAP-2301

Build : http://builds.cask.co/browse/CDAP-RT19

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java [hunk 1]
```diff
@@ -31,7 +27,8 @@
 public class RealtimeStageContext implements StageContext {
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java]: how come this class is needed if there is a WorkerRealtimeContext or vice versa?

Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java]: o is this is for transforms? is the rename happening in a separate PR?

**Comment by albertshau** [comment]
one question, if rename happening later, lgtm

File: cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java (hunk 1 above)
Comment (gokulavasan) [inline cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java]: Yes its for Transform. Renaming of Transform related classes will happen in a separate PR.

**Comment by gokulavasan** [comment]
Yes, renaming happening later. Will merge. Thanks.

**Pull request merged**